GME/Gme/GME.rc

```diff
@@ -1131,14 +1156,15 @@ BEGIN
     CONTROL         "Save credentials",IDC_CHECK_PERMANENT,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,7,84,69,10,WS_EX_RIGHT
 END
 
+
 /////////////////////////////////////////////////////////////////////////////
 //
 // Version
 //
 
 VS_VERSION_INFO VERSIONINFO
- FILEVERSION GME_VERSION_RC
- PRODUCTVERSION GME_VERSION_RC
+ FILEVERSION 11,11,3,0

```

Comment:
Please revert this unintentional changes

Reply:
Done, did you notice any others? I found one, and updated it as well.